jax/api.py

```diff
@@ -28,6 +28,8 @@
 import functools
 import inspect
 import itertools as it
+import os
+import string
```

Comment:
I think these two imports are unused. (Easy to fix later!)

Reply:
Removed!